object_manager.go

```diff
@@ -413,7 +450,7 @@ func (objMgr *ObjectManager) GetHostRecord(ref string) (*HostRecord, error) {
 	return recordHost, err
 }
 
-func (objMgr *ObjectManager) GetHostRecordWithoutDNS(recordName string, netview string, cidr string, ipAddr string) (*HostRecord, error) {
+func (objMgr *ObjectManager) GetHostRecordWithoutref(recordName string, netview string, cidr string, ipAddr string) (*HostRecord, error) {
```

Comment:
i feel `GetHostRecordByName` would  be better. May be change the above to `GetHostRecordWithRef`

Reply:
okay ,will do it